src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -302,14 +304,13 @@ void AcousticWaveEquationSEM::computeSeismoTrace( real64 const time_n,
                                                   arrayView1d< real64 const > const var_n,
                                                   arrayView2d< real64 > varAtReceivers )
 {
-  real64 const time_np1 = time_n+dt;
+  real64 const time_np1 = time_n + dt;
   arrayView2d< localIndex const > const receiverNodeIds = m_receiverNodeIds.toViewConst();
   arrayView2d< real64 const > const receiverConstants   = m_receiverConstants.toViewConst();
   arrayView1d< localIndex const > const receiverIsLocal = m_receiverIsLocal.toViewConst();
 
-  real64 const a1 = (dt < epsilonLoc) ? 1.0 : (time_np1 - timeSeismo)/dt;
+  real64 const a1 = (abs( dt ) < epsilonLoc) ? 1.0 : (time_np1 - timeSeismo)/dt;
```

Comment:
`dt` is always positive, can't give negative values for `dt`. See `SolverBase::execute()` method

Reply:
in backward we give -dt in parameter.